internal/it/util.go

```diff
@@ -542,3 +588,68 @@ func sortedString(b []byte) string {
 	s = strings.ReplaceAll(s, "\r", "")
 	return s
 }
+
+var nextPort int32 = 10000
+
+func NextPort() int {
+	// let minimum step be 10, just a round, safe value.
+	// note that some tests may not use the MemberCount() function for the cluster size.
+	const maxStep = 10
+	step := MemberCount()
+	if step < maxStep {
+		step = maxStep
+	}
+nextblock:
+	for {
+		start := int(atomic.AddInt32(&nextPort, int32(step))) - step
```

Comment:
There is no limit check for `65535`. int32 can exceed it.

Reply:
We'll immediately know when that happens, so that's not very important.